internal/commands/report.go

```diff
@@ -82,8 +83,19 @@ Config:
 }
 
 func sanitize(line string) string {
-	if strings.HasPrefix(line, "default-builder-image") {
-		return `default-builder-image = "[REDACTED]"`
+	re := regexp.MustCompile(`"(.*?)"`)
```

Comment:
Did this in order to preserve the number of values in arrays so that we can determine if some bugs show up with mutiple values.

A test was added for `mirrors` that uses multiple values.